.circleci/config.yml

```diff
@@ -321,41 +337,6 @@ commands:
             done;
           #
 
-  conditionally-launch-build-and-store:
```

Comment:
Steps combined into `app-build` job.
In this command, it launches a separate CircleCI workflow to do the build for a changed study. Triggered workflow is `build-app-workflow`. Because it's a separate workflow, it's not easy to find out status of new workflow: when it has started, passed or failed. Only way to find out workflow status is by calling other CircleCI api which would make config much more complicated.